.github/configs/mr-test-example.yaml

```diff
@@ -0,0 +1,32 @@
+## Example configuration
+#################### syntax #################
+## include:
+##   - dataset: ["<dataset_name>"]
+##     config: ["<configuration_name>"]
+#
+## Example:
+## include:
+##  - dataset: ["Carbon Bot"]
+##    config: ["Sparse + DIET(bow) + ResponseSelector(bow)"]
+#
+## Shortcut:
+## You can use the "all" shortcut to include all available configurations or datasets
+#
+## Example: Use the "Sparse + EmbeddingIntent + ResponseSelector(bow)" configuration
+## for all available datasets
+## include:
+##  - dataset: ["all"]
+##    config: ["Sparse + DIET(bow) + ResponseSelector(bow)"]
+#
+## Example: Use all available configurations for the "Carbon Bot" and "Sara" datasets
+## and for the "Hermit" dataset use the "Sparse + DIET + ResponseSelector(T2T)" and
+## "Sparse + ConveRT + DIET + ResponseSelector(T2T)" configurations:
+## include:
+##  - dataset: ["Carbon Bot", "Sara"]
+##    config: ["all"]
+##  - dataset: ["Hermit"]
+##    config: ["Sparse + DIET(seq) + ResponseSelector(t2t)", ["Sparse + ConveRT + DIET(seq) + ResponseSelector(t2t)"]
```

Comment:
that's not valid yaml right? Is the second `[` needed?
```suggestion
##    config: ["Sparse + DIET(seq) + ResponseSelector(t2t)", "Sparse + ConveRT + DIET(seq) + ResponseSelector(t2t)"]
```

Reply:
Good eye 👍